canvas/services/event.py

```diff
@@ -52,6 +44,29 @@ def get_question_stats(question):
                 answers[answer] = 0
             answers[answer] += 1
 
+    users = []
+    progress_submission_statuses = [0, 0, 0]
+    student_submissions = submissions.filter(uqj__user__role="Student")
+    correct_submissions = student_submissions.filter(is_correct=True).values("uqj__user_id")
```

Comment:
`.values_list("uqj__user_id", flat=True)` returns just the ids in an array, instead of a dictionary with the `"uqj__user_id"` key, would save you from have to do things like `correct["uqj__user_id"]`

Reference: https://stackoverflow.com/questions/37205793/django-values-list-vs-values